### PR TITLE
Public 2490 dylan 20230728

### PR DESCRIPTION
**Pull request by dcchivian**
JIRA RELEASE TICKET: https://kbase-jira.atlassian.net/browse/SCT-3223
BETA TEST NARRATIVE: https://narrative.kbase.us/narrative/92659

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,8 +22,12 @@ RUN make all
 # Install Gblocks
 #
 WORKDIR /kb/module
+
+# GONE
+#    curl http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_Linux64_0.91b.tar.Z > Gblocks_Linux64_0.91b.tar.Z && \
+
 RUN \
-    curl http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_Linux64_0.91b.tar.Z > Gblocks_Linux64_0.91b.tar.Z && \
+    cp tarballs/Gblocks_Linux64_0.91b.tar.Z . && \
```
Comment (psdehal) [inline Dockerfile]: The lab moved and so the URL is wrong, it should be:
https://www.biologiaevolutiva.org/jcastresana/Gblocks/Gblocks_Linux64_0.91b.tar.Z

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+### Version 1.0.8
+- handle long gene IDs
+- use own copy of tarball since original gone from web
```
Comment (psdehal) [inline RELEASE_NOTES.md]: There's an updated URL you could have used, but I think this is fine. I don't think it's under active development anymore and it's easy enough to include the tarball.

File: ui/narrative/methods/run_Gblocks/display.yaml [hunk 3]
```diff
@@ -88,7 +88,7 @@ parameters :
             If first row of MSA has "X" masking characters, remove those positions (columns) from output MSA; Usually used for hypervariable regions (e.g. 16S); OFF=default.
 
 description : |
-    <p>Gblocks 0.91b from http://molevol.cmima.csic.es/castresana/Gblocks.html</p>
+    <p>Gblocks 0.91b was originally downloaded from http://molevol.cmima.csic.es/castresana/Gblocks.html.  The server no longer responds, so this App uses a copy that is retained in this KBase module at https://github.com/kbaseapps/kb_gblocks/blob/master/tarballs/Gblocks_Linux64_0.91b.tar.Z</p>
```
Comment (psdehal) [inline ui/narrative/methods/run_Gblocks/display.yaml]: There's a new URL but I think this is fine

**Review by psdehal — APPROVED** [review]
There's two things, neither of them do you have to fix. First, there is a new URL https://www.biologiaevolutiva.org/jcastresana/Gblocks/Gblocks_Linux64_0.91b.tar.Z

it's up to you if you want to use the URL or the tarball, I don't they're actively working on GBlocks, so the tarball is marginally more robust to a hosting change. Second, it would have been fine to delete the existing commented out code.
No change requests.

File: Dockerfile (hunk 1 above)
Comment (dcchivian) [reply Dockerfile]: made update

File: RELEASE_NOTES.md (hunk 2 above)
Comment (dcchivian) [reply RELEASE_NOTES.md]: made update

File: ui/narrative/methods/run_Gblocks/display.yaml (hunk 3 above)
Comment (dcchivian) [reply ui/narrative/methods/run_Gblocks/display.yaml]: made update

**Pull request merged**